src/userLayout/Checkout.tsx

```diff
@@ -53,16 +63,19 @@ const PayBooking = () => {
         <form
           onSubmit={handleSubmit}
           style={{
-            width: "30rem",
+            width: "40rem",
             background: "#bbf7d0",
             padding: "3rem",
             border: "none",
             borderRadius: "2rem",
+            marginTop: "5em",
           }}
         >
           <AddressElement options={{ mode: "billing" }} />
           <CardElement />
-          <button>Submit</button>
+          <Button sx={{ width: "100%" }} type="submit" variant="contained">
```

Comment:
We can disable this button when the stripe instance is still loading, we can also change the text to "book" instead of "submit"
`disabled={!stripe}`